src/constants.js

```diff
@@ -0,0 +1,9 @@
+export const TRIP_STATUS = {
```

Comment:
can we name this file something a bit more descriptive like we do our other const files? https://github.com/radarlabs/radar-sdk-js/blob/master/src/error_codes.js